sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -2006,6 +2006,7 @@ SIGVERIFY_BAD_CASES = [
 
 test_suite(
     name = "sigverify_always",
+    tags = ["manual"],
```

Comment:
nit: this is inline with what we decided to do but this test_suite doesn't have to be tagged manual since it doesn't depend on bitstream_splice, right?

Reply:
Test suites should almost always be tagged manual so they don't break *_tag_filters. This one can cause a bunch of problems without it though you're right it's also messing up the test along with the other unexpected behaviors:

`bazel build //... -build_tag_filters=-cw310` will build all the cw310 tests in this suite because this test suite isn't tagged cw310. If this includes non-cw310 tests it shouldn't be tagged cw310 or it will exclude the other tests, so really we just want this suite to be manual.

 This suite depends on all the tests in it and transitively everything they depend on, so it does depend on a bitstream_splice, but because it's not tagged with "cw310_rom" it doesn't get the benefit of the exception I wrote for tests that depend on that.

I should probably just write a query for test_suites that aren't tagged manual before executing any other ones, but that would belong in another PR.

Reply:
I should fix the vivado query to exclude bitstream splices that are dependencies of the cached bitstreams